config/fam_metadata_config.yaml

```diff
@@ -18,4 +18,11 @@ metadata_path: /dev/shm/
 # where interface address is hostname (fqdn)/ip4 address where grpc port should be exposed.
 # portno is port to be used by OpenFAM for communication between Metadata Manager and
 # Client Interface Service.
-rpc_interface: 127.0.0.1:8989
+rpc_interface: 127.0.0.1:8788
+
+# Flag to enable or disable region spanning. By default, it is set to true.
+enable_region_spanning: true
+
+#If region spanning is enabled, obtain the value fo size_per_memoryserver.
+size_per_memoryserver=1048576
```

Comment:
Let the default value be larger - i.e 1 GB or so. Also typo in "obtain the value fo size_per_memoryserver" .